libdeflate.h

```diff
@@ -262,6 +262,19 @@ libdeflate_zlib_decompress(struct libdeflate_decompressor *decompressor,
 			   void *out, size_t out_nbytes_avail,
 			   size_t *actual_out_nbytes_ret);
 
+/*
+ * Like libdeflate_zlib_decompress(), but adds the 'actual_in_nbytes_ret'
+ * argument.  If 'actual_in_nbytes_ret' is not NULL and the decompression
+ * succeeds, then the actual number of input bytes consumed is written to
```

Comment:
This isn't clear about when the decompression stops.  Can you mention that it decompresses the first zlib-compressed stream (similar to the comment for `libdeflate_gzip_decompress_ex()`)?:

```
/*
 * Like libdeflate_zlib_decompress(), but adds the 'actual_in_nbytes_ret'
 * argument.  If 'actual_in_nbytes_ret' is not NULL and the decompression
 * succeeds (indicating that the first zlib-compressed stream in the input
 * buffer was decompressed), then the actual number of input bytes consumed is
 * written to *actual_in_nbytes_ret.
 */
```

It would also be helpful to update the comment for `libdeflate_zlib_decompress()` to clarify that it stops at the end of the zlib stream:

```
/*
 * Like libdeflate_deflate_decompress(), but assumes the zlib wrapper format
 * instead of raw DEFLATE.
 *
 * Decompression will stop at the end of the zlib stream, even if it is shorter
 * than 'in_nbytes'.  If you need to know exactly where the zlib stream ended,
 * use libdeflate_zlib_decompress_ex().
 */

```